tools/anvil-join-anvil

```diff
@@ -1589,9 +1589,9 @@ sub wait_for_access
 			}});
 			
 			### NOTE: I know I could match interface names, but that's not certain enough. It's 
-			###       possible (if unlikely) that the network name+numbre differs on our peer. So
+			###       possible (if unlikely) that the network name+number differs on our peer. So
 			###       this is safer.
-			# Loop through my peer's interfaces and see if we're sharing this one.
+			# Loop through my peer's interfaces and see if we're shareing this one.
```

Comment:
`shareing`->`sharing`

Reply:
English is my first language, good lord...